app/config.py

```diff
@@ -9,12 +12,57 @@ def get_json_dict(path):
     try:
         with open(path) as file:
             return json.load(file)
-    except:
+    except Exception as e:
+        print("Error on path ", path, e)
```

Comment:
Flask commands now complain about a missing `config/events.json` file. I think we might want to add a blank `events.json` to avoid this error.

Reply:
Nice, thanks!